fe1-web/res/strings.ts

```diff
@@ -99,13 +99,15 @@ const STRINGS = {
   election_create_setup: 'Election Setup',
   election_create_name: 'Name*',
   election_create_start_time: 'Start time: ',
-  election_create_finish_time: 'End time: ',
+  election_create_finish_time: 'End time:  ',
```

Comment:
is the extra space on purpose?

Reply:
Yes, I tried to make the election setup look a tiny bit better. 
![image](https://user-images.githubusercontent.com/28142119/116421925-90c27080-a83f-11eb-9c75-886cb11c990f.png)
With the extra space, the datepicker lookes more centered. But this could probably also be solved with styling.